src/component/ChairCard/ChairCard.js

```diff
@@ -30,36 +34,47 @@ export default function ChairCard({
           ></ButtonDescription>
         </div>
       </ChairDetails>
+      <InspirationImg
+        inspirationImg1={inspirationImg1}
+        inspirationImg2={inspirationImg2}
+        inspirationImg3={inspirationImg3}
+      />
     </ChairItem>
   );
 }
 
 const ChairItem = styled.section`
   display: flex;
   flex-flow: column wrap;
-  margin: 1rem;
-  gap: 1rem;
+  margin: 0.5rem;
```

Comment:
Why did you changed your gap into a margin? 
You can use `gap: 1rem` to add some space between flex-items - one of the cool flex-box use cases 😊